### PR TITLE
Feature: Add support for base mainnet

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task ID: [OS-634](https://aragonassociation.atlassian.net/browse/OS-634)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them when possible.
- [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the `[UPCOMING]` title and before the latest version.
- [x] I have tested my code on the test network.

[OS-634]: https://aragonassociation.atlassian.net/browse/OS-634?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/client-common/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client-common",
   "author": "Aragon Association",
-  "version": "1.3.0-rc0",
+  "version": "1.4.0-rc0",
```
Comment (heueristik) [inline modules/client-common/package.json:4]: This does not seem correct.
Base support is not a code feature of the SDK as this PR does not introduce new functionality.  It is a "meta" change. 
So the right number should be 1.3.0-rc0.2 in my opinion to signal that.

File: modules/client-common/CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ TEMPLATE:
 
 ## [UPCOMING]
 ### Added
+- Support for baseMainnet network
+## 1.3.0-rc0
+### Added
 - Support for baseGoerli network
```
Comment (heueristik) [inline modules/client-common/CHANGELOG.md:24]: As mentioned in https://github.com/aragon/sdk/pull/265/files#r1282826187, this might not be correct.

Comment (josemarinas) [reply modules/client-common/CHANGELOG.md:24]: nope, this one is correct, I added the previous version to the changelog

File: modules/client-common/package.json (hunk 1 above)
Comment (heueristik) [reply modules/client-common/package.json:4]: After discussion, this seems to be correct for the SDK.

**Review by heueristik — APPROVED** [review]

**Pull request merged**